README.md

```diff
@@ -156,6 +156,23 @@ $ bundle exec ruby benchmarks.rb
 If your results differ from what's posted here,
 [let us know](https://github.com/panorama-ed/memo_wise/issues/new)!
 
+## Thread Safety
+
+MemoWise makes the following **thread safety** guarantees on all supported Ruby
+versions:
+
+1. **Before** a value has been memoized
+
+   * Contended calls from multiple threads...
+      * May call original method multiple times
```

Comment:
I was a bit confused by this wording. Maybe something like:

* May each call the original method

?

Reply:
Changed!